src/inputs/radio/radio.jsx

```diff
@@ -156,6 +165,12 @@ const styles = (theme) => {
                     paddingTop: 2,
                 },
             },
+            '& .radio-required-indicator': {
+                color: theme.color(theme.palette.error.main),
+                display: 'inline-block',
+                fontSize: theme.typography.pxToRem('14px'),
```

Comment:
I think you can do `theme.typography.pxToRem(14)` but not sure if it matters much (if at all)

Reply:
Fixed: https://github.com/saddlebackdev/react-cm-ui/pull/418/commits/d5596bc2248dbcbca6e36727cfb2c110c79e1006#diff-11845ddfa7f03b6ae429b010969b1f4bf254fad4da3580a55075f42beaf6db25L171-R172